generic-tabs/GenericTabs.js

```diff
@@ -35,11 +35,19 @@ export class GenericTabs extends SelectedMixin(BatchingElement) {
     return {
       selectors: {
         tabs: {
-          selector: el => el.querySelectorAll("[slot='tab']"),
+          selector: el =>
+            Array.from(el.children).filter(node =>
+              node.matches('h1, h2, h3, h4, h5, h6, [slot="tab"]'),
```

Comment:
should this include [role="heading"][aria-level] as a selector too, do you think? unsure.

Reply:
For now, my preference would be to not include that yet, in order to limit the scope of this change, and prevent other potential issues from being surfaced. For instance, I think this library adds the role of tab to the lightdom, which would require that role attribute to be managed if, say, the heading were pushed into or pulled out of the tab list.

Reply:
sgtm. Hey I just noticed this issue is #42  pe heading tabs really are the answer to life, the universe, and everything